core/src/main/java/com/jexunit/core/spi/BeforeSheet.java

```diff
@@ -0,0 +1,6 @@
+package com.jexunit.core.spi;
+
```

Comment:
Could you please add some explaining documentation?

Reply:
I'll fix that.